templates/readme.md

```diff
@@ -0,0 +1,61 @@
+# DOT Graph template
+
+```
+crane template -t path/to/graph.tmpl -o output/myGraph.dot
+```
+
+Generate a [DOT](http://en.wikipedia.org/wiki/DOT_(graph_description_language)) file representing the dependency graph. Bold nodes represent the containers declared in the config (as opposed to non-bold ones that are referenced in the config, but not defined). Targeted containers are highlighted with color borders. Solid edges represent links, dashed edges volumesFrom, and dotted edges net=container relations.
+
+# Systemd template
```

Comment:
couldn't these templates be tested from `template_test.go` to make sure they remain valid no matter what we do (horrible conscious breaking changed or other) ?